docs/docs/add-offline-support-with-a-service-worker.md

```diff
@@ -41,6 +41,8 @@ Add this plugin to your `gatsby-config.js`
 
 That's all you need to add offline support to your Gatsby site.
 
+Note: You must create a production build using `gatsby build` for the service workers to register.
```

Comment:
Maybe something like (just to avoid words like `must` but still convey the message)
```suggestion
Note: Service worker registers only in production builds (`gatsby build`).
```